resources/config.edn

```diff
@@ -28,31 +28,9 @@
 
  :fluree/subscriptions {}
 
- :fluree/standalone {:address #or [#env FLUREE_STANDALONE_ADDRESS
-                                   #profile {:dev    "/ip4/127.0.0.1/tcp/62071"
-                                             :prod   nil
-                                             :docker "/ip4/127.0.0.1/tcp/62071"}]}}
- :fluree/raft {:log-history      #or [#env FLUREE_RAFT_LOG_HISTORY
```

Comment:
I think we want to preserve these settings somewhere, maybe a second config-raft.edn file to at least have them captured.

Reply:
Yes, either in a separate config raft file, docs, or?

I was planning on starting another branch for config stuff to see how far I get with implementing your suggestions from yesterday.

Reply:
I've saved the raft config options in a separate file. We can merge the config map in that file with the main one, and then `dissoc` the `:fluree/standalone` key and start raft normally. I will wait until we implement the more user friendly configs to make that process more ergonomic.